### ISSUE TITLE
Undo - threat model diagram editor

### ISSUE THREAD

**Issue by kparris007** [mike-goodwin/owasp-threat-dragon#156]
Undo button or Ctrl-Z does not work in the threat model diagram editor.

**Comment by jgadsden** [comment]
Yes, agreed - it is something that we would like to add to TD and is the subject of an issue in the migrated repo :

'unable to Undo' - https://github.com/OWASP/threat-dragon-core/issues/4

There is a discussion on whether to migrate to draw.io or stay with jointjs - draw.io would provide the undo/redo straight away
